tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -566,27 +566,25 @@ def zip(datasets):
     For example:
 
     ```python
-    # NOTE: The following examples use `{ ... }` to represent the
-    # contents of a dataset.
-    a = { 1, 2, 3 }
-    b = { 4, 5, 6 }
-    c = { (7, 8), (9, 10), (11, 12) }
-    d = { 13, 14 }
+    a = Dataset.range(1, 4)  # [ 1, 2, 3 ]
```

Comment:
minor consistency nit: elsewhere you use "==> in the comment", here you don't

this comment applies throughout your PR

Reply:
Fixed this one in several places in the latest commit